iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -547,14 +547,15 @@ export class Viewer {
      *
      * Takes an optional list of factIds corresponding to all facts that a click
      * falls within.  If omitted, it's treated as a click on a non-nested fact.
+     *
+     * byClick indicates that the element was clicked directly, and in this
+     * case we never scroll to make it more visible.
      */
-    selectElement(itemId, itemIdList) {
+    selectElement(itemId, itemIdList, byClick) {
         if (itemId !== null) {
-            this.onSelect.fire(itemId, itemIdList);
-        }
-        else {
```

Comment:
Can you explain the removal of the `else` here? Why do we want to call `this.onSelect.fire(null);` unconditionally?

Reply:
@paulwarren-wk

Reply:
This was a mistake.  I somehow imagined a `return` inside the `if`.  Reverted in 13a36c3